float64[] activations
---
bool success
